crm-project/src/main/java/CRM/controller/controllers/UserController.java

```diff
@@ -43,4 +43,11 @@ public ResponseEntity<Response> getAllInBoard(@PathVariable Long boardId){
         Response response = userFacade.getAllInBoard(boardId);
         return new ResponseEntity<>(response, response.getStatus());
     }
+
+    @PostMapping(value = "add/{userId}/toBoard/{boardId}")
+    public ResponseEntity<Response> addUserToBoard(@PathVariable Long userId, @PathVariable Long boardId){
+
```

Comment:
delete line

Reply:
I did it on purpose, as validating the data is a different code block for me.
To delete it?
@shailevi23